atomicapp/run.py

```diff
@@ -55,6 +52,7 @@ def __init__(
             answers_format=ANSWERS_FILE_SAMPLE_FORMAT, **kwargs):
 
         self.debug = debug
+        self.display = Display()
```

Comment:
I'll unify this.

For some reason, if you have display() initialized from within the `main.py/install.py/run.py` classes, it grabs `logging.getLogger("atomicapp").isEnabledFor(logging.DEBUG)` correctly, but if it's outside of the class, it doesn't. For the life of me I couldn't fix it.

Once we have the unified base class from #145 we can just grab from the base class whether self.verbose is set or not.